# Cliff (drop-off) sensor states
std_msgs/Header header

bool left_side      # Left side cliff detected
bool left_front     # Left front cliff detected
bool right_front    # Right front cliff detected
bool right_side     # Right side cliff detected
